src/topology/constructions.lean

```diff
@@ -64,6 +64,15 @@ instance Pi.topological_space {β : α → Type v} [t₂ : Πa, topological_spac
 instance ulift.topological_space [t : topological_space α] : topological_space (ulift.{v u} α) :=
 t.induced ulift.down
 
+lemma is_open.quotient_preimage [topological_space α] [s : setoid α]
+  {V : set $ quotient s} (h : is_open V) : is_open (quotient.mk ⁻¹' V) :=
+is_open_preimage_of_coinduced rfl h
+
+lemma quotient.preimage_mem_nhds [topological_space α] [s : setoid α]
+  {V : set $ quotient s} {q : quotient s}
+  {a : α} (hab : quotient.mk a = q) (hs : V ∈ 𝓝 q) : quotient.mk ⁻¹' V ∈ 𝓝 a :=
```

Comment:
On the same theme as the one below, this lemma could eliminate `q` as
```suggestion
  {a : α} (hs : V ∈ 𝓝 (quotient.mk a)) : quotient.mk ⁻¹' V ∈ 𝓝 a :=
```

Reply:
This will mean an extra rewrite when using the lemma, but it's true that it makes the statement cleaner, so let's do that.